### PR TITLE
Shops and Products in Db

### PR DESCRIPTION
**Pull request by fluriumteam**
Fix

### CODE REVIEW

File: Data/Models/Product.cs [hunk 1]
```diff
@@ -13,19 +13,28 @@ public class Product
...
         public string PreviewImage { get; set; }
-        public List<ProductImage> Images { get; set; }
+        public IReadOnlyCollection<ProductImage> Images { get; set; }
 
-        //public List<CategoryToProduct> Categories?
         public string ShopId { get; set; }
-
         public Shop Shop { get; set; }
-
+        public bool isDraft { get; set; } = false;
```
Comment (roman-koshchei) [inline Data/Models/Product.cs:20]: IsDraft = true

File: Data/Models/Product.cs [hunk 2]
```diff
@@ -13,19 +13,28 @@ public class Product
...
         public string? VideoUrl { get; set; }
 
         //SEO
         public string SeoTitle { get; set; } = string.Empty;
 
         public string SeoDescription { get; set; } = string.Empty;
         public string SeoSlug { get; set; } = string.Empty;
+
+        public Product() { }
```
Comment (roman-koshchei) [inline Data/Models/Product.cs:29]: Remove empty constructors

**Review by roman-koshchei — CHANGES_REQUESTED** [review]
Remove empty constructors. We create constructors exactly to avoid empty.

File: Data/Models/Product.cs [hunk 3]
```diff
@@ -17,17 +17,14 @@ public class Product
 
         public string ShopId { get; set; }
         public Shop Shop { get; set; }
-        public bool isDraft { get; set; } = false;
+        public bool isDraft { get; set; } = true;
```
Comment (fluriumteam) [inline Data/Models/Product.cs:20]: с большой IsDraft

**Review by roman-koshchei — APPROVED** [review]

**Pull request merged**